### ISSUE TITLE
Sovereign Tech fund - proposal for nbviewer

### ISSUE THREAD

**Issue by trallard** [jupyter/governance#172]
Hey, y'all - I am sorry I am opening this issue so late.

TLD;R @Carreau  and I are working on a proposal for the [Sovereign Tech Fund - Three challenges to contribute back to OS](https://sovereigntechfund.de/en/challenges/) within the `Tooling` category.
Details are as follows:

1. Who is involved in the proposal?
@Carreau and I are writing this proposal and submitting it through Quansight Labs where we are both employed. I have also consulted with @minrk to buy some of his time in an advisory/reviewer role (i.e. architectural design, review PR, brainstorming) through his current employer.

3. What projects will it target? Do you know what the scope is?

* [nbviewer](https://github.com/jupyter/nbviewer)
* [nbviewer-deploy](https://github.com/jupyter/nbviewer.org-deploy)

Despite being a critical project within the Jupyter ecosystem nbviewer is overdue for maintenance and upgrades. Additionally, we'd like to add the following:

* Support for JHub 2.0 authentication
* [CI improvements around deployment](https://github.com/jupyter/nbviewer.org-deploy) (akin to what is being used for mybinder.org) will also reduce the burden on a single individual (@minrk, who has been a while the primary/solely responsible for updates and deployments)

4. Logistics:
* We are working on the proposal (due the 6th of July - this month), as per the RFD this covers about 4 months of work
* Once submitted, we will share the proposal publicly as we have done in the past (for example, the Jupyter accessibility CZI grant)
* If funded, and similar to the Jupyter a11y grant, we will manage the grant and its deliverable in public through GitHub for transparency among the broader community and accountability

**Comment by jasongrout** [comment]
Thanks for the notification about this proposal! Since it appears that your proposal is not in Jupyter’s name, nor using the Jupyter or NumFOCUS financial accounts, the EC does not need to approve this. We encourage you to work with the maintainers of nbviewer and the @jupyter-software-steering-council (which is the [governing council](https://jupyter.org/governance/list_of_subprojects.html#official-subprojects-without-ssc-representation) for this repo) to make sure this proposal aligns with the direction of the subproject.

- Jason Grout, on behalf of the Executive Council

**Comment by trallard** [comment]
Thanks for the clarification @jasongrout 🙏🏻, as I went through the governance docs at https://jupyter.org/governance/list_of_subprojects.html#notes-about-official-jupyter-subprojects I assumed that since this work touches both nbviewer and work towards the improvement of nbviewer-deploy workflows this would be a suitable avenue to raise such an effort.
For transparency we have liaised with the team to align priorities for the project and buy some of their time for PR reviews, feedback and such. 

Will drop a line to the SSC as suggested (and close this issue to avoid noise).
